test/libsolidity/util/TestFileParser.cpp

```diff
@@ -92,7 +92,10 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 						if (lowLevelCall)
 							call.kind = FunctionCall::Kind::LowLevel;
 						else if (isBuiltinFunction(call.signature))
+						{
 							call.kind = FunctionCall::Kind::Builtin;
+							call.builtinReturnType = m_builtins.at(call.signature).returnType;
```

Comment:
Please store the type in the expectations instead.

Reply:
I _think_ you have to do this not at the point of parsing but after expectation failure, similar to
```
					abiParams = ContractABIUtils::parametersFromJsonOutputs(
						_errorReporter,
						m_contractABI,
						m_call.signature
					);
```